### PR TITLE
mv pagerduty2slack to schedules2slack

### PR DESCRIPTION
**Pull request by geisslet**

### CODE REVIEW

File: global/schedules2slack/Chart.lock [hunk 1]
```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: owner-info
+  repository: https://charts.eu-de-2.cloud.sap
+  version: 0.9.3
```
Comment (IvoGoman) [inline global/schedules2slack/Chart.lock:4]: This version does not exists: https://github.com/sapcc/helm-charts/blob/master/common/owner-info/Chart.yaml
```suggestion
  version: 0.2.3
```

File: global/schedules2slack/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: schedules2slack
+version: 0.0.1
+description: Tool for syncing servicenow schedules with slack handles
+dependencies:
+  - name: owner-info
+    repository: https://charts.eu-de-2.cloud.sap
+    version: 0.9.3
```
Comment (IvoGoman) [inline global/schedules2slack/Chart.yaml:8]:
```suggestion
    version: 0.2.3
```

**Review by IvoGoman — CHANGES_REQUESTED** [review]
Besides the dependency version this LGTM

**Review by geisslet — COMMENTED** [review]
thanks!

**Pull request merged**